source/core/ut_suite_cache_manager.pks

```diff
@@ -55,11 +55,14 @@ create or replace package ut_suite_cache_manager authid definer is
   * Not to be used publicly. Used internally for building suites at runtime.
   */
   function get_cached_suite_rows(
+    a_suites_filtered ut_suite_cache_rows 
+  ) return ut_suite_cache_rows;
+
+  function get_cached_suites(
```

Comment:
We need to check where it is used.
If usage is broad, it may be easier to keep the name `get_cached_suites`  and rename the `get_suite_items` in the body

Reply:
Used only in one place. is good to remove.